plugins/automox/plugin.spec.yaml

```diff
@@ -0,0 +1,988 @@
+      title: Device Limit
+      description: The organization device limit
+      type: integer
+      required: false
+    device_count:
+      title: Device Count
+      description: The organization device count
+      type: integer
+      required: false
+  users:
+    id:
+      title: User ID
+      description: The user identifier
+      type: integer
+      required: true
+    firstname:
+      title: First Name
+      description: The first name of the user
+      type: string
+      required: false
+    lastname:
+      title: Last Name
+      description: The last name of the user
+      type: string
+      required: false
+    email:
+      title: Email
+      description: The email of the user
+      type: string
+      required: false
+    features:
+      title: Features
+      description: The features enabled for the user
+      type: "object"
+      required: false
+    orgs:
+      title: Organizations
+      description: The organizations for which the user has access
+      type: "[]object"
+      required: false
+    tags:
+      title: Tags
+      description: The user defined tags
+      type: "[]string"
+      required: false
+    saml_enabled:
+      title: SAML Enabled
+      description: Whether SAML has been enabled for the user
+      type: boolean
+      required: false
+    rbac_roles:
+      title: Roles
+      description: The roles assigned to the user
+      type: "[]object"
+      required: false
+  device:
+    id:
+      title: Device ID
+      description: The device ID
+      type: integer
+      required: true
+    server_group_id:
+      title: Server Group ID
+      description: The server group identifier of the device
+      type: integer
+      required: true 
+    organization_id:
+      title: Organization ID
+      description: The organization identifier of the device
+      type: integer
+      required: true
+    uuid:
+      title: Device UUID
+      description: The device unique identifier
+      type: string
+      required: true
+    name:
+      title: Device Name
+      description: The device name
+      type: string
+      required: true
+  policy:
+    id:
+      title: Policy ID
+      description: The policy identifier
+      type: integer
+      required: true
+    organization_id:
+      title: Organization ID
+      description: The organization identifier of the policy
+      type: integer
+      required: true
+    name:
+      title: Policy Name
+      description: The name of the policy
+      type: string
+      required: false
+    policy_type_name:
+      title: Policy Type Name
+      description: The type of policy
+      type: string
+      required: false
+    configuration:
+      title: Policy Configuration
+      description: The configuration of the policy
+      type: object
+      required: false
+    server_groups:
+      title: Device Groups
+      description: List of device groups assigned policy
+      type: "[]integer"
+      required: false
```

Comment:
Is this a typo? Field name is server_groups, title/description is device groups. Also please review the data type here- is every device group an integer?

Reply:
I'm updating the description to be a bit more clear - along with the title. It's a list of Device Group IDs.